src/Uno.UI/UI/Xaml/Media/VisualTreeHelper.cs

```diff
@@ -96,7 +96,7 @@ public static int GetChildrenCount(DependencyObject reference)
 #endif
 		}
 
-		public static IReadOnlyList<IPopup> GetOpenPopups(Window window)
+		internal static IReadOnlyList<IPopup> GetOpenPopups(Window window)
```

Comment:
? This is part of the UWP contract

Reply:
IPopup is Uno's stuff.

Reply:
But you're right. <https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.visualtreehelper.getopenpopups> It's just not exactly the right signature.

Reply:
Fixed.